app/bundles/AssetBundle/Model/AssetModel.php

```diff
@@ -124,6 +124,12 @@ public function trackDownload($asset, $request = null, $code = '200', $systemEnt
             $request = $this->requestStack->getCurrentRequest();
         }
 
+        if (!($request instanceof Request)) {
```

Comment:
```suggestion
        if (!($request instanceof RequestInterface)) {
```

Reply:
The Symfony Request class does not implement RequestInterface, so this check will never be valid.

Reply:
You're right!